.eslintrc.json

```diff
@@ -53,7 +53,11 @@
       ],
       "rules": {
         "react-hooks/rules-of-hooks": "error", // Checks rules of Hooks
-        "react-hooks/exhaustive-deps": "warn" // Checks effect dependencies
+        "react-hooks/exhaustive-deps": "warn", // Checks effect dependencies
+        "react/jsx-first-prop-new-line": "error",
```

Comment:
Do we have the react plugin for eslint already installed? I could not find it in the package.json.s

Reply:
good find: https://github.com/filecoin-station/filecoin-station/pull/465